nbextensions/upload_it/main.js

```diff
@@ -0,0 +1,111 @@
+/**
+ * @license
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+define([
+  'jquery',
+  'base/js/namespace',
+  'base/js/dialog',
+], function($, Jupyter, dialog) {
+  "use strict";
+
+  // Default values for the configuration parameters.
+  var configuration = {
```

Comment:
`let` and `const` are preferred in JavaScript.
https://google.github.io/styleguide/jsguide.html#features-use-const-and-let

Though, It's not the case if the backward compatibility is important.

Reply:
We do not know for sure how old browser we should support, but I suppose it is okay to just target recent Chrome.
Changed all var to let or const.